src/index.html

```diff
@@ -1,19 +1,20 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>To-Do List</title>
-    <link rel="stylesheet" href="css/styles.css" />
-  </head>
-
-  <body>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task Master</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
     <div class="container">
-      <h1>To-Do List</h1>
-      <input type="text" id="taskInput" placeholder="Add a new task" />
-      <button id="addTaskBtn">Add Task</button>
-      <ul id="taskList"></ul>
+        <h1>Task Master</h1>
+        <div class="task-input">
+            <input type="text" id="taskInput" placeholder="Add a new task...">
+            <button id="addTaskBtn">Add Task</button>
+        </div>
+        <ul id="taskList"></ul>
     </div>
-    <script src="js/script.js"></script>
-  </body>
+    <script src="script.js"></script>
```

Comment:
Can you keep the location of the js file the same as before? It does not live in the same directory as this file.